bioblend/_tests/TestGalaxyInvocations.py

```diff
@@ -172,6 +175,24 @@ def test_rerun_invocation(self):
         history = self.gi.histories.show_history(rerun_invocation["history_id"], contents=True)
         assert len(history) == 3
 
+    @test_util.skip_unless_galaxy("release_24.02")
+    def test_rerun_invocation_with_input_params(self):
+        threeline_dataset_id = self._test_dataset(self.history_id, contents="A\nB\nC")
+        invocation = self._invoke_x_random_lines_workflow(threeline_dataset_id)
+        self.gi.invocations.wait_for_invocation(invocation["id"])
+        params = {
+            "how_many": {
+                "parameter_value": "1",
+                "label": "how_many",
+            }
+        }
+        rerun_invocation = self.gi.invocations.rerun_invocation(
+            invocation["id"], params_update=params, history_id=self.history_id
```

Comment:
```suggestion
            invocation["id"], params_update=params
```

Reply:
This will fix the test, but not the issue. It should be possible to specify a history, let me investigate.